compiler/p/codegen/ControlFlowEvaluator.cpp

```diff
@@ -2757,17 +2757,20 @@ static void lookupScheme1(TR::Node *node, bool unbalanced, bool fromTableEval, T
          int64_t value = child->getCaseConstant();
          if (cg->comp()->target().is64Bit())
             {
+            // TODO: handle value out of immediate range
```

Comment:
Is there a plan for dealing with these TODOs? Can an issue be opened to track this work?

Reply:
Yes, an issue will be opened.

Reply:
Removed the TODOs since as @aviansie-ben pointed out there is no functional issue here. Opened issue https://github.com/eclipse/omr/issues/5323 to improve the code.